docs/TranslatableFields.md

```diff
@@ -9,33 +9,35 @@ You may have fields which are translated in multiple languages and want users to
 
 ```js
 {
-    name: {
-        en: 'The english value',
-        fr: 'The french value',
-        tlh: 'The klingon value',
+    title: {
+        en: 'Doctors Without Borders',
+        fr: 'Médecins sans frontières',
     },
     description: {
-        en: 'The english value',
-        fr: 'The french value',
-        tlh: 'The klingon value',
-    }
+        en:
+            'International humanitarian medical non-governmental organisation of French origin',
+        fr:
+            "Organisation non gouvernementale (ONG) médicale humanitaire internationale d'origine française fondée en 1971 à Paris",
+    },
 }
 ```
 
-This is how to use it:
+Use `<TranslatableFields>` like so:
 
 ```jsx
 <TranslatableFields locales={['en', 'fr']}>
-    <TextField source="name" />
+    <TextField source="title" />
     <TextField source="description" />
 </TranslatableFields>
 ```
 
+![TranslatableFields addLabel](./img/TranslatableFields.png)
```

Comment:
I'd put this screenshot front and foremost in the page, probably after the initial sentence.